src/ecs/commands/runCommandInContainer.ts

```diff
@@ -187,3 +146,58 @@ async function verifyCliAndPlugin(window: Window): Promise<boolean> {
     }
     return true
 }
+
+async function isDeploymentInProgress(node: EcsContainerNode): Promise<boolean> {
+    const deployments = (await node.ecs.describeServices(node.clusterArn, [node.serviceName]))[0].deployments
+    if (deployments) {
+        for (let i = 0; i < deployments.length; i++) {
+            if (deployments[i].status === 'ACTIVE' || deployments[i].rolloutState === 'IN_PROGRESS') {
+                return true 
+            }
+        }
+    }
+    return false
+}
+
+async function getTaskItems(node: EcsContainerNode): Promise<vscode.QuickPickItem[]> {
+    const taskArns = await node.listTasks()
+    // Filter for only 'Running' tasks
+    const runningTasks = (await node.describeTasks(taskArns)).filter(t => {
+        return t.lastStatus === 'RUNNING' && t.desiredStatus === 'RUNNING'
+    })
+    return runningTasks.map(task => {
+        // The last 32 digits of the task arn is the task identifier
+        return {
+            label: task.taskArn!.substr(-32),
+            detail: `Status: ${task.lastStatus}  Desired status: ${task.desiredStatus}`,
+        }
+    })
+}
+
+async function runCliCommandInEcsContainer(node: EcsContainerNode, task: string, command: string, outputChannel: vscode.OutputChannel) {
+    const cmd = new ChildProcess(
+        true,
+        'aws',
+        undefined,
+        'ecs',
+        'execute-command',
+        '--cluster',
+        node.clusterArn,
+        '--task',
+        task,
+        '--container',
+        node.containerName,
+        '--command',
+        command,
+        '--interactive'
+    )
+    const processResponse = await cmd.run()
+    if (processResponse.exitCode !== 0) {
+        showOutputMessage(processResponse.stderr, outputChannel)
```

Comment:
Just making sure, is there any output from stdout in an error case or just stderr?